pkgs/development/beam-modules/rebar3-release.nix

```diff
@@ -48,7 +49,7 @@ let
     self: stdenv.mkDerivation (attrs // {
 
       name = "${name}-${version}";
```

Comment:
if `pname` is provided I believe we can omit this

Reply:
```suggestion
      name = "${pname}-${version}";
```

Reply:
Thank you for the feedback